dq-server/src/db.js

```diff
@@ -98,8 +98,10 @@ const questionResultSchema = new mongoose.Schema({
 			type: mongoose.Schema.Types.ObjectId,
 			ref : 'Player',
 		},
+		hasAnswer: Boolean,
```

Comment:
What does this do

Reply:
@manuelaguirre

Reply:
represents player has answered the question

Reply:
hadAnswered: boolean is better